[msx] testStarted testDFSSeekExceptions
2019-12-19 15:37:47,065 [main] INFO  namenode.NameNode (NameNode.java:<init>(946)) - [msx-restart] reconfigMode is none, do nothing
2019-12-19 15:37:47,848 [main] INFO  namenode.NameNode (NameNode.java:<init>(1016)) - [msx-restart] NameNode start HostAndPort localhost:36126
2019-12-19 15:37:47,918 [main] INFO  datanode.DataNode (DataNode.java:<init>(464)) - [msx-restart] DataNode new conf
2019-12-19 15:37:48,395 [main] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2653)) - [msx-restart] DataNode start
2019-12-19 15:37:48,400 [main] INFO  datanode.DataNode (DataNode.java:<init>(464)) - [msx-restart] DataNode new conf
2019-12-19 15:37:48,464 [main] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2653)) - [msx-restart] DataNode start
2019-12-19 15:37:49,565 [main] INFO  datanode.DataNode (DataNode.java:shutdown(1995)) - [msx-restart] DataNode stop
2019-12-19 15:37:49,634 [main] INFO  datanode.DataNode (DataNode.java:shutdown(1995)) - [msx-restart] DataNode stop
2019-12-19 15:37:49,791 [main] INFO  namenode.NameNode (NameNode.java:stop(1059)) - [msx-restart] NameNode stop HostAndPort localhost:36126 double check before stop dfs.namenode.audit.log.async = false
[msx] testFinished testDFSSeekExceptions testresult=1
